### PR TITLE
Add dataset.ini config file for model finetuning

### PR DESCRIPTION
**Pull request by the-sink**
Changes `download_reddit_finetuning_data.py` and `output_finetuning_data.py` to use config variables assigned in `dataset.ini` as opposed to hard-coded parameters in the scripts themselves (#16)

### CODE REVIEW

**Review by zacc — CHANGES_REQUESTED** [review]

File: model_finetuning/dataset_template.ini [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[DEFAULT]
+
+; REQUIRED, a comma-separated list of subreddits for Pushshift to download from
+subreddits=test
```
[the remark this answers is not in the document]
Comment (the-sink) [reply model_finetuning/dataset_template.ini]: Fixed (424c5ff)

File: model_finetuning/download_reddit_finetuning_data.py [hunk 2]
```diff
@@ -102,14 +103,33 @@ def main():
...
 	submission_limit = 100
 
+	# pull configs from dataset.ini
+	if config['DEFAULT']['start_date']:
+		start_date = config['DEFAULT']['start_date']
+	if config['DEFAULT']['end_date']:
+		end_date = config['DEFAULT']['end_date']
+	if config['DEFAULT']['subreddits']:
+		subreddits = config['DEFAULT']['subreddits'].split(',')
+	if config['DEFAULT']['submission_limit']:
```
[the remark this answers is not in the document]
Comment (the-sink) [reply model_finetuning/download_reddit_finetuning_data.py:124]: Not sure how I missed that... 424c5ff

File: model_finetuning/download_reddit_finetuning_data.py [hunk 3]
```diff
@@ -102,14 +103,33 @@ def main():
...
+		submission_limit = config['DEFAULT']['submission_limit']
```
[the remark this answers is not in the document]
Comment (the-sink) [reply model_finetuning/download_reddit_finetuning_data.py]: Fixed (424c5ff)

**Review by zacc — APPROVED** [review]
Nice work!

**Pull request merged**